lap-mysql-ubuntu/azuredeploy.json

```diff
@@ -0,0 +1,262 @@
+﻿{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "region": {
+      "type": "string",
+      "allowedValues": [
```

Comment:
VMs can now be deployed in almost all regions. Consider removing the allowed values list.